sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -548,6 +548,54 @@ def test_binomial_rewrite():
     assert binomial(n, k).rewrite(ff) == ff(n, k) / factorial(k)
     assert binomial(n, x).rewrite(ff) == binomial(n, x)
 
+def test_multinomial():
+    x = Symbol('x')
+    n = Symbol('n', integer=True)
+    nz = Symbol('nz', integer=True, nonzero=True)
+    k = Symbol('k', integer=True)
+    kp = Symbol('kp', integer=True, positive=True)
+    kn = Symbol('kn', integer=True, negative=True)
+    u = Symbol('u', negative=True)
+    v = Symbol('v', nonnegative=True)
+    p = Symbol('p', positive=True)
+    z = Symbol('z', zero=True)
+    nt = Symbol('nt', integer=False)
+    kt = Symbol('kt', integer=False)
+    a = Symbol('a', integer=True, nonnegative=True)
+    b = Symbol('b', integer=True, nonnegative=True)
```

Comment:
```suggestion
    x = Symbol('x')
    n, k = symbols('n, k', integer=True)
    nz  = Symbol('nz', integer=True, nonzero=True)
    kp = Symbol('kp', integer=True, positive=True)
    kn = Symbol('kn', integer=True, negative=True)
    u = Symbol('u', negative=True)
    v = Symbol('v', nonnegative=True)
    p = Symbol('p', positive=True)
    z = Symbol('z', zero=True)
    nt, kt = symbols('nt, kt', integer=False)
    a, b = symbols('a, b', integer=True, nonnegative=True)
```
Not compulsory to apply but puts variables with same assumptions together.

Reply:
gotcha ... it looks better too !

Reply:
I'll remove unused variables ... I'll add em as I extend functionality 😄